scalyr_agent/util.py

```diff
@@ -77,8 +77,8 @@ def ujson_dumps_custom(obj, fp):
             :param obj: The object to serialize
             :param fp: If not None, then a file-like object to which the serialized JSON will be written.
             :type obj: dict
-            :return: If fp is not None, then the str representing the serialization.
-            :rtype: str
+            :return: If fp is not None, then the string representing the serialization.
+            :rtype: Python3 - six.text_type, Python2 - six.text_type | six.binary_type
```

Comment:
Add documentation as to under what conditions `text_type` or `binary_type` are returned in Python 2.

Reply:
I have to admit that I have been a bit hasty. Only standard library `json.dumps` with the option `ensure_ascii=False` can produce `unicode`, so, unless we do not use such option, it can return only six.binary_type